io.github.babaissarkar.ssplot.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id": "io.github.babaissarkar.ssplot",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": "org.freedesktop.Sdk.Extension.openjdk17",
+    "command": "ssplot",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host:rw"
```

Comment:
why host filesystem permissions?

Reply:
So that it can open and save files.

Reply:
read plot data and write the resultant image back

Reply:
this doesn't justify `host` at all.

Reply:
is `--filesystem=home:rw` okay?

Reply:
changed to xdg-documents